### DISCUSSION TITLE
Cleanup unused VRFs?

### DISCUSSION THREAD

**Discussion by jbemmel** [ipspace/netlab#778] [General]
The vxlan-centralized-irb test cases creates an empty vrf on leaf2, at least on SR Linux.
Do we want to filter it out?

i.e. the VRF is associated with 2 VLANs, but both are mode=bridge so there is no IP functionality required

**Comment by ipspace** [comment]
> The vxlan-centralized-irb test cases creates an empty vrf on leaf2, at least on SR Linux. Do we want to filter it out?

There were several reasons to have VRFs on nodes where they were not strictly needed, but I can't remember what they were. EVPN RD/RT?

**Comment by ipspace** [comment]
Nothing has moved for a year. It probably wasn't too critical ;)
